code/modules/mob/living/simple_animal/hostile/angel_statue.dm

```diff
@@ -180,33 +180,27 @@
 	return
 
 //Blind AOE
-/obj/effect/proc_holder/spell/aoe/blindness
+/obj/effect/proc_holder/spell/blindness
 	name = "Blindness"
 	desc = "Your prey will be momentarily blind for you to advance on them."
 
 	message = "<span class='notice'>You glare your eyes.</span>"
 	base_cooldown = 600
 	clothes_req = FALSE
-	aoe_range = 10
 
-/obj/effect/proc_holder/spell/aoe/blindness/create_new_targeting()
-	var/datum/spell_targeting/aoe/turf/targeting = new()
-	targeting.range = aoe_range
-	return targeting
+/obj/effect/proc_holder/spell/blindness/create_new_targeting()
+	return new /datum/spell_targeting/self
 
-/obj/effect/proc_holder/spell/aoe/blindness/cast(list/targets, mob/user = usr)
-	for(var/mob/living/L in GLOB.alive_mob_list)
-		if(L == user)
+/obj/effect/proc_holder/spell/blindness/cast(list/targets, mob/user = usr)
+	for(var/mob/living/M in orange(10, user))
```

Comment:
Ideally, you keep this an AOE spell, and just iterate through targets, and just check at they're living, and not a weeping angel